### DISCUSSION TITLE
Gear - Involute Rack

### DISCUSSION THREAD

**Discussion by leyhans** [looooo/freecad.gears#171] [Q&A]
Hello,
I have used **"Gear"** to design different types of gearwheels without any problems.
Today I have tried to use **"Involute Rack"** for the first time and it doesn't function or I don't understand it.
I got only the message **"cannot pickle 'Part.Wire' object"**

**Comment by looooo** [comment]
I can't reproduce. Maybe you are using an older version of this addon.

**Comment by leyhans** [comment]
Thanks for your answer. I have tried different times to delete and to install new by Addon-Manger. A friend had the same problem. Today I tried again and it looks OK! Thanks a lot!!
